02-express-tutorial/app.js

```diff
@@ -50,14 +76,35 @@ app.get("/api/v1/query", (req, res) => {
   res.json(filteredProducts);
 });
 
-app.get("/", (req, res) => {
-  res.sendFile(path.resolve(__dirname, "./public/index.html"));
+
+app.get("/api/v1/people", (req, res) => {
+  res.json(people);
+});
+
+
+app.post("/api/v1/people", (req, res) => {
+  if (!req.body.name) {
+    return res.status(400).json({ success: false, message: "Please provide a name" });
+  }
+
+  
+  const newPerson = {
+    id: people.length + 1,  
+    name: req.body.name,   
+  };
+
+  people.push(newPerson);
+
+  
+  res.status(201).json({ success: true, name: req.body.name });
 });
```

Comment:
the refactoring is not quite complete here. this is where we're just wanting to use the `peopleRouter` to replace the inline code

Reply:
I replaced the initial /api/v1/people routes (such as app.post, app.get, etc.) with the peopleRouter to handle all these routes and ensure better modularity and code organization.